data/data/gcp/main.tf

```diff
@@ -66,6 +65,10 @@ module "network" {
   bootstrap_instances = module.bootstrap.bootstrap_instances
 
   master_instances = module.master.master_instances
+
+  preexisting_cluster_network = var.gcp_vpc
+  preexisting_master_subnet   = var.gcp_control_plane_subnet
+  preexisting_worker_subnet   = var.gcp_compute_subnet
```

Comment:
Can we not just pull these from network/output.tf and set them there based on inputs?

Reply:
We need to specify these here as inputs to network, but we can use network/output.tf for the rest of the modules. That seemed to work pretty well and was cleaner than I anticipated.